Vagrantfile

```diff
@@ -34,14 +36,24 @@ Vagrant.configure("2") do |config|
     cartodb.vm.network "private_network", ip: ENV.fetch("CARTODB_PRIVATE_IP", "192.168.20.100")
 
     cartodb.vm.synced_folder ".", "/vagrant", disabled: true
-    cartodb.vm.synced_folder ENV.fetch("CARTODB_SRC_DIR", "../cartodb"), "/opt/cartodb", type: "nfs", mount_options: MOUNT_OPTIONS
+    cartodb.vm.synced_folder ENV.fetch("CARTODB_SRC_DIR", "../cartodb"), ROOT_VM_DIR, type: "nfs", mount_options: MOUNT_OPTIONS
 
-    cartodb.vm.provision "ansible" do |ansible|
+    cartodb.vm.provision "bootstrap", type: "ansible" do |ansible|
       ansible.playbook = "ansible/cartodb.yml"
       ansible.groups = ANSIBLE_GROUPS.merge(ANSIBLE_ENV_GROUPS)
       ansible.raw_arguments = ["--timeout=60"]
     end
 
+    # Restart some cartodb services on each boot
+    # Needs to be after ansible provision and also in a block definition so this runs after ansible
+    cartodb.vm.provision "restart_services", type: "shell", run: "always" do |s|
+      s.inline = "service cartodb_server restart && service cartodb_resque_worker restart WORKERNUM=1"
```

Comment:
So the issue here is that the upstart job doesn't properly start these two services when the vm boots. I tried a few different start_on definitions, but couldn't get anythign to work. So this is  a workaround. I didn't add to ansible because we want these to trigger on every vm boot, not just when ansible is run.